app/models/pet.rb

```diff
@@ -10,4 +13,9 @@ def shelter_name
   def self.adoptable
     where(adoptable: true)
   end
+
+  def add_pet_to_application(pet_id)
```

Comment:
This is implemented in 2 different models -  no need for it to live in more than one place! These are also things you could do in a `create` action of your pet_applications_controller.rb